### PR TITLE
Polish step and repl

### PR DESCRIPTION
**Pull request by Yancey0623**
TODO: using `step` in workflow e2e test. https://github.com/sql-machine-learning/sqlflow/issues/1746

### CODE REVIEW

File: cmd/step/main_test.go [hunk 1]
```diff
@@ -82,3 +85,47 @@ func TestStepTrainSQL(t *testing.T) {
...
+
+func TestStepStandardSQL(t *testing.T) {
+	if os.Getenv("SQLFLOW_TEST_DB") != "mysql" {
+		t.Skip("skip no mysql test.")
+	}
+	a := assert.New(t)
+	dbConnStr := "mysql://root:root@tcp(127.0.0.1:3306)/iris?maxAllowedPacket=0"
+	session := makeTestSession(dbConnStr)
+	sql := `SELECT * FROM iris.train limit 5;`
+	a.NoError(checkStepWrapper(func() { a.NotPanics(func() { runSQLStmt(sql, session) }) }, func(s string) error {
```
Comment (typhoonzero) [inline cmd/step/main_test.go]: Maybe just reuse `getStdout` in `repl_test.go`, passing two functions here is hard to read.

**Comment by tonyyang-svail** [comment]
One general question. Looks like there is some code duplication between `pakcage repl` and `package step`. Can we expose a function `RunSQLProgramAndPrintResult` in `package step` and then make `repl` depend on this function?

**Comment by Yancey0623** [comment]
> One general question. Looks like there is some code duplication between pakcage repl and package step. Can we expose a function RunSQLProgramAndPrintResult in package step and then make repl depend on this function?

@tonyyang-svail Good point I will do that.

File: cmd/repl/repl.go [hunk 2]
```diff
@@ -11,13 +11,13 @@
...
 
-package main
+package repl
 
 import (
 	"bufio"
 	"bytes"
 	"image"
-	_ "image/png"
+	_ "image/png" // comment(yancey1989): why import this blank import
```
Comment (shendiaomo) [inline cmd/repl/repl.go]: Using the side effect to register functions that processes `png` is a typical usage of the `image` module.

**Review by shendiaomo — COMMENTED** [review]
Should we put the `main` function in a `main` package?

File: cmd/repl/repl.go (hunk 2 above)
Comment (Yancey0623) [reply cmd/repl/repl.go]: I moved the common code to `pkg/step`, and add some tests.

**Review by typhoonzero — APPROVED** [review]

**Pull request merged**